pkg/crc/preflight/preflight.go

```diff
@@ -92,9 +96,19 @@ func (check *PreflightCheck) doFix() error {
 	return check.fix()
 }
 
+func (check *PreflightCheck) doCleanUp() error {
+	if check.cleanupDescription == "" {
+		panic(fmt.Sprintf("Should not happen, empty description for cleanup '%s'", check.configKeySuffix))
+	}
```

Comment:
@cfergeau for cleanup, we should not allow skip option otherwise the left over still present and user again report back to us.

Reply:
The problem is that we cannot know why the 'skip' config option was set. Maybe it's because they did not want crc to make any changes related to that option in the first place, so trying to cleanup these changes is wrong, and could even mess up the system, as we'd be changing something we did not setup in the first place.

Reply:
@cfergeau So as of now we are only changing something which CRC do as part of preflight checks even if those are skip and all those are very specific to `CRC` what do you think we are cleaning something which a user can set manually and as part of `crc cleanup` we are removing it ?

Reply:
The comment above does not match what the code is doing, but more on this below

Reply:
@cfergeau we do check first if file is present but if you very strong about adding the skip functionality here I can add but I don't see any use for this in `cleanup` right now.

Reply:
Right now, I'm trying to understand how you want this to work, I'm not yet feeling strongly about any options :)
Are you saying that the 'cleanup' methods should ignore as many errors as possible when doing their job, to cope with both the nominal case, and cases where some of the checks are disabled? Which would mean we don't need to check the config options, as the 'cleanup' method will be appropriate in both cases. Or do you have something different in mind?

Reply:
> Are you saying that the 'cleanup' methods should ignore as many errors as possible when doing their job, to cope with both the nominal case, and cases where some of the checks are disabled?

We will only ignore the error if something is not present, not just ignore errors which need to be processed, Yes I think `cleanup` shouldn't depend on the what changes a user `skip` as part of setup we never know if this is first time setup for the user or something he skipped recently and we are not removing that because of it is skipped in config.